roles/awx/files/awx-deployment.yml

```diff
@@ -0,0 +1,133 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: awx-deployment
+parameters:
+- name: NAMESPACE
+- name: AWX_WEB_IMAGE
+- name: AWX_TASK_IMAGE
+- name: AWX_ADMIN_PASSWORD
+
+objects:
+
+- kind: Secret
+  apiVersion: v1
+  metadata:
+    name: admin-credentials
+    namespace: ${NAMESPACE}
+  stringData:
+    username: admin
+    password: ${AWX_ADMIN_PASSWORD}
+
+- apiVersion: extensions/v1beta1
+  kind: Deployment
+  metadata:
+    name: awx
+    namespace: ${NAMESPACE}
+  spec:
+    replicas: 1
+    template:
+      metadata:
+        labels:
+          name: awx-web-deploy
+          service: django
+      spec:
+        containers:
+          - name: awx-web
+            image: ${AWX_WEB_IMAGE}
+            ports:
+              - containerPort: 8052
+            volumeMounts:
+              - mountPath: /etc/tower
+                name: awx-application-config
+            env:
+              - name: DATABASE_PASSWORD
+                valueFrom:
+                  secretKeyRef:
+                    name: postgresql
+                    key: database-password
+          - name: awx-celery
+            image: ${AWX_TASK_IMAGE}
+            volumeMounts:
+              - mountPath: /etc/tower
+                name: awx-application-config
+            env:
+              - name: DATABASE_USER
+                value: awx
+              - name: DATABASE_NAME
+                value: awx
+              - name: DATABASE_HOST
+                value: postgresql
+              - name: DATABASE_PORT
+                value: "5432"
+              - name: DATABASE_PASSWORD
```

Comment:
Environment is not a secure place for secrets... can pgsql consume the password from a file on disk?